docs/how_to/query.md

```diff
@@ -0,0 +1,47 @@
+# Setting up m3query
+
+## Introduction
+
+m3query is used to query data that is stored in M3DB. For instance, if you are using the Prometheus remote write endpoint with [m3coordinator](../integrations/prometheus.md), you can use m3query instead of the Prometheus remote read endpoint. By doing so, you get all of the benefits of m3query's engine. Furthermore, since m3query provides a Prometheus compatible API, you use 3rd party graphing and alerting solutions like Grafana.
+
+<!-- TODO: link to m3query docs -->
+
+## Configuration
+
+Before setting up m3query, make sure that you have at least [one M3DB node running](single_node.md). In order to start m3query, you need to configure a `yaml` file, that will be used to connect to M3DB. Here is a link to a [sample config](https://github.com/m3db/m3/blob/master/src/query/config/m3query-local-etcd.yml) file that is used for an embedded etcd cluster.
```

Comment:
We have a Dockerfile for `m3query` right? Should we suggest using that?

Reply:
We do, but I think it just uses a default config file so not sure we should recommend using it.

Reply:
Looks like it's using this exact config https://github.com/m3db/m3/blob/master/docker/m3query/Dockerfile#L24 . That said, we should probably make sure it works before recommending it haha. In the meantime, can we provide specific instructions on building on running? e.g.

```
make m3query
./bin/m3query -f ./src/query/config/m3query-local-etcd.yml
```

Reply:
+1 andrew and then also you say that the config is used for an embedded etcd cluster but thats kind of confusing because its the m3db node that runs the embedded etcd cluster